ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml

```diff
@@ -1862,3 +1862,85 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  labels:
+    job-type: launch
+    job-env: openstack
+    job-release: '4.2'
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  interval: 4h
+  name: canary-openshift-ocp-installer-e2e-openstack-4.2
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --secret-dir=/usr/local/pull-secret
```

Comment:
Hmm, the usual OpenStack presubmit [only has the profile secret][1].  I don't understand how that works, but it seems like [it does][2].  Maybe try to get closer to that template, or try to get @flaper87 to explain why he didn't need the pull-secret entry in #1824?

[1]: https://github.com/openshift/release/blob/7c02339ca3cc0e4a6c1f05ad4c7854d5d8c1fd49/ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml#L991
[2]: https://ci-search-ci-search-next.svc.ci.openshift.org/chart?name=^pull-.*-openstack&search=fatal.*waiting%20for%20Kubernetes%20API:%20context%20deadline%20exceeded

Reply:
Could be related to the old "inline" nature of the secrets. I pushed this to align OpenStack with the latest attachements: https://github.com/openshift/release/pull/4823

Reply:
One more observation would be what you said above applies to all the AWS jobs too. OpenStack is working in presubmit, yes! But it seems like the Canary jobs should align as well.